web/packages/teleport/src/Integrations/status/AwsOidc/AwsOidcDashboard.tsx

```diff
@@ -87,6 +117,11 @@ export function AwsOidcDashboard() {
             summary={awseks}
           />
         </Flex>
+        {listStatus === 'success' && !enrolledCli && (
+          <Flex>
+            <ConsoleCardEnroll />
+          </Flex>
+        )}
```

Comment:
regarding the AWSOIDC dashboard, in the un-enrolled state, i see this section in this dashboard that says enable access to console access, but afterwards, this section goes away.

i almost expected it to be the same section there, but in a enabled state, and either lead me to that integration or lead me to a list of console access (the resources page)

just double checking that was intentional

Reply:
This is intentional! There will be follow up work to enhance the flow but this is the MVP.